proto/gz/msgs/camerasensor.proto

```diff
@@ -24,21 +24,88 @@ option java_outer_classname = "CameraSensorProtos";
 /// \interface CameraSensor
 /// \brief Information about a camera sensor element
 
-import "gz/msgs/vector2d.proto";
 import "gz/msgs/distortion.proto";
+import "gz/msgs/double.proto";
 import "gz/msgs/header.proto";
+import "gz/msgs/lens.proto";
+import "gz/msgs/sensor_noise.proto";
+import "gz/msgs/vector2d.proto";
 
 message CameraSensor
 {
+  /// \brief Bounding box types.
+  enum BoundingBoxType
+  {
+    /// \brief No bounding box.
+    NO_BOUNDING_BOX = 0;
+
+    /// \brief 2D box that shows the full box of occluded objects
+    FULL_BOX_2D     = 1;
+
+    /// \brief 2D box that shows the visible part of the occluded object
+    VISIBLE_BOX_2D  = 2;
+
+    /// \brief 3D oriented box
+    BOX_3D          = 3;
+  }
+
+  /// \brief Segmentation types.
+  enum SegmentationType
+  {
+    /// \brief No segmentation.
+    NO_SEGMENTATION = 0;
+
+    /// \brief Pixels of same label from different items
+    /// have the same color & id.
+    SEMANTIC        = 1;
+
+    /// \brief Pixels of same label from different items, have different
+    /// color & id. 1 channel for label id & 2 channels for instance id
+    PANTOPIC        = 2;
+  }
+
   /// \brief Optional header data
   Header header         = 1;
 
+  /// \brief Horizontal field of view
   double horizontal_fov = 2;
+
+  /// \brief Image size in pixels.
   Vector2d image_size   = 3;
+
+  /// \brief Image format
```

Comment:
How about using `PixelFormatType`?

https://github.com/gazebosim/gz-msgs/blob/39bbcdfc8b6312c882f0e38d28c5c43f0784b8db/proto/gz/msgs/image.proto#L29-L52

Reply:
This field has existed since at least Citadel. Changing it without a tick-tock would be dangerous.

Reply:
Added `pixel_format` as an alternative in [08dac4b](https://github.com/gazebosim/gz-msgs/pull/201/commits/08dac4b4cbceefd536234fedfb3a08e7b1ec7c3d)